api/data-source/data-source.ts

```diff
@@ -82,5 +84,10 @@ const addOrder = (order: any) => {
   return order;
 };
 
-export const ordersMock = { orders, addOrder };
-export const eventsMock = { events, addEvent, updateEvent };
+const updateOrder = (order: any, id: string, index: string | number) => {
+  (<any>orders)[index] = { id, ...order };
+  return (<any>orders)[index];
+};
+
+export const ordersDataSource = { orders, addOrder, updateOrder };
+export const eventsDataSource = { events, addEvent, updateEvent };
```

Comment:
export `find` instead of `events`

```js
// find :: () -> IEvent[]
const find = () => events;

// ...

export const eventsDataSource = { find, addEvent, updateEvent };
```